packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ReactElement } from 'react';
+import { useState } from 'react';
+import styled from 'styled-components/native';
+import type { BaseInputProps } from './BaseInput';
+import getBaseInputStyles from './getBaseInputStyles';
+import type { Theme } from '~components/BladeProvider';
+
+export const StyledNativeBaseInput = styled.TextInput(
+  (props: BaseInputProps & { isFocussed: boolean; theme: Theme }) => ({
```

Comment:
Should add the props in generic instead of overriding the argument. 

```
export const StyledNativeBaseInput = styled.TextInput<BaseInputProps & { isFocussed: boolean }>(
  (props) => ({
```

Reply:
+1 Lets do this ^

Reply:
fixed in later PRs